experimental/dds/tree2/src/feature-libraries/chunked-forest/chunkTree.ts

```diff
@@ -211,7 +236,7 @@ function tryShapeForSchema(
  *
  * Note that this does not tolerate optional or sequence fields, nor does it optimize for patterns of specific values.
  */
-function tryShapeForFieldSchema(
+export function tryShapeForFieldSchema(
```

Comment:
nit: I feel like this isn't a great name. Maybe `tryGetShapeFromFieldSchema`?

Reply:
I feel like "get" implies getting something with already exists. Given that thus function is passed in a map which it could look up the result in, I think get would imply thats all it does.

We could go with "getOrCreate", but I choose to model it more after the naming conventions for conversion style functions ( xFromY ) and add try since it can fail. Looking at the names though, I was not at all consistent about that. I'll fix it.

Reply:
now all the functions related to this use "From".